app/models/prior_authority_application.rb

```diff
@@ -34,11 +34,11 @@ class PriorAuthorityApplication < ApplicationRecord
     draft: 'draft',
     submitted: 'submitted',
     granted: 'granted',
-    part_granted: 'part_granted',
+    part_grant: 'part_grant',
```

Comment:
So there shouldn't _be_ any legacy data, because there was never a way of _assigning_ a `part_granted` status - the caseworker app only ever set `part_grant`. But just in case I added a migration in case in some environment, for reasons unknown, someone manually set the status of a record to `part_granted`.